taxi_service/settings.py

```diff
@@ -123,6 +123,7 @@
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
 STATIC_URL = "static/"
+STATICFILES_DIRS = [BASE_DIR / "static"]
```

Comment:
The 'STATICFILES_DIRS' setting is correctly configured to include the 'static' directory. Ensure that this directory exists in your project structure.